pal/src/exception/machmessage.h

```diff
@@ -55,6 +55,21 @@ typedef	mach_exception_data_type_t	mach_exception_subcode_t;
 
 struct MachExceptionHandler;
 
+class MachMessage;
+
+// Contains all the exception and thread state information needed to forward the exception.
```

Comment:
This is also pulled from latest PAL: https://github.com/dotnet/coreclr/blob/775003a4c72f0acc37eab84628fcef541533ba4e/src/pal/src/exception/machmessage.h#L86, commit https://github.com/dotnet/coreclr/commit/9f6654b6faec5730f71d6c547ea497e406ae4d47, dated 1/26/2016.

The 3 `.S` assembly files are same as PAL master.